setup.py

```diff
@@ -25,14 +25,19 @@ def find_version(*file_paths):
 version = find_version("gpytorch", "__init__.py")
 
 
-torch_min = "1.9"
-install_requires = [">=".join(["torch", torch_min]), "scikit-learn", "scipy"]
+torch_min = "1.11"
```

Comment:
Is PyTorch 1.11 a requirement to properly implement the LinearOperator functionality? Or can we deal with 1.10 also?

The current requirements would mean skipping the 1.10 release as a min requirement. This is a pretty aggressive version requirement that could make it hard for some people in production settings to use this.

It's a major update of gpytorch and so it's fine, but if we can avoid it then that may make sense.

Reply:
The new linear_operator package makes use of some 1.11 naming conventions/helper methods.

Though linear_operator could easily be downgraded, I think the best solution is: 1) cut a release before we merge this PR in, where we bump up the version to 1.10. Then 2) when we merge this PR, we bump to 1.11.